docs/team/ansenn.md

```diff
@@ -0,0 +1,53 @@
+# Ansenn - Project Portfolio Page
+
+### Project: PENUS
+PENUS is a desktop app for NUS engineering students to manage and plan their modules in their university life. The user interacts with it using a CLI. It is written in Java, and has about 6 kLoC.
+
+Given below are my contributions to the project.
+
+- **Feature** : List command `list`
+    - What it does: Displays a list of all modules taken or planned in a specified Year and/or Semester.
+    - Highlights : Has a flexible format to its inputs with 3 different variations, which made it harder to parse the user's input and check for exceptions.
+
+- **Feature** : Help Command `help`
+    - What it does: Displays the list of accepted commands, and each command's purpose and use.
+
+- **Feature**: Clear Command `clear`
+    - What it does: Clears all modules added to the planner in a specified Year and/or Semester.
+    - Highlights : Similar to the `list` command, the `clear` command has a flexible format to its inputs with 3 different variations, which made it harder to parse the user's input and check for exceptions.
+
+- **Enhancement** : Changed the PENUS logo 
+  - Improved the logo of PENUS
+
+- **Code contributed**: [RepoSense link](https://nus-cs2113-ay2223s2.github.io/tp-dashboard/?search=ansenn&breakdown=true)
+
+- **Contributions to UG**:
+  - Edit formatting of various commands in the UG
+  - Improved clarity of the UG in response to PE-D comments 
+  - Written Documentation for:
+    - List Command
+    - Help Command
+    - Clear Command
+  - Sequence diagram for Clear Command
```

Comment:
Perhaps sequence diagram should be under DG instead?

Reply:
I believe it can be under the (Extracts) for his PPP?

Reply:
Dear Jiun Yuan and Sriram,

I hope this comment finds you well. I just wanted to take a moment to thank you for taking the time to review my pull request. Your feedback and suggestions have been incredibly helpful in improving my code, and I truly appreciate your efforts.

Your expertise and attention to detail are invaluable, and I am grateful for the opportunity to work with you. Please let me know if there is anything else I can do to improve the code, or if you have any further suggestions or feedback.

Once again, thank you for your help and support. I look forward to continuing to work with you.

Best regards,
Ansenn